task_processing/plugins/mesos/execution_framework.py

```diff
@@ -374,15 +378,6 @@ def statusUpdate(self, driver, update):
         if update.state == 'TASK_FINISHED':
             self.task_metadata.pop(task_id, None)
 
-        if update.state in (
```

Comment:
my mistake, i only needed to remove the `task_update_queue.put` line because we already enqueue that update in line 374. good catch!